test/salt/pillar/exim.sls

```diff
@@ -3,8 +3,13 @@
 ---
 exim:
   config:
-    configtype: 'satellite'
+    configtype: satellite
     hide_mailname: 'false'
```

Comment:
You probably want to make `hide_mailname` value as boolean  too.

Reply:
> You probably want to make `hide_mailname` value as boolean too.

I want to test that string values and boolean values will be casted in the same way.

Reply:
Sounds reasonable!